docs/src/main/sphinx/connector/clickhouse.rst

```diff
@@ -90,13 +92,13 @@ Property Name               Default Value    Description
 =========================== ================ ==============================================================================================================
 ``engine``                  ``Log``          Name and parameters of the engine.
 
-``order_by``                (none)           list of columns to be the sorting key. Required if ``engine`` is ``MergeTree``.
+``order_by``                (none)           Array of columns or expressions to use as the sorting key. Required if ``engine`` is ``MergeTree``.
 
-``partition_by``            (none)           list of columns to be the partition key. Optional.
+``partition_by``            (none)           Array of columns or expressions to use as the partition key. Optional.
 
-``primary_key``             (none)           list of columns to be the primary key. Optional.
+``primary_key``             (none)           Array of columns or expressions to use as the primary key. Optional.
 
-``sample_by``               (none)           An expression for sampling. Optional.
+``sample_by``               (none)           An expression to use for sampling. Optional.
```

Comment:
Wording is okay .. but what does that mean? If this is used for table stats .. then maybe use

An expression to use for sampling data for table statistics generation.

Reply:
It allows you to perform the equivalent of `TABLESAMPLE` in ClickHouse. See https://clickhouse.tech/docs/en/sql-reference/statements/select/sample/.

i.e. it's useful to set if someone is accessing the table from ClickHouse.

Reply:
Lets add a link to that then..

Reply:
Added. Please take a look.